### ISSUE TITLE
Gamma filter

### ISSUE THREAD

**Issue by devmarco** [h2non/bimg#270]
Does Bimg support the `Gamma` filter?

I'm trying to migrate a project from `Nodejs` using `sharp` to `Go` using `bimg`, and the question about gamma is because Sharp has support for it and I have been using that. http://sharp.pixelplumbing.com/en/stable/api-operation/#gamma

**Comment by h2non** [comment]
No, it does not, but it can be supported in the future, since both libraries rely on `libvips`.

**Comment by sugimotoren** [comment]
Hello @h2non, I'd like to work on this issue.
Could you please give me any advice on my PR?
